tutorial/source/appendix.rst

```diff
@@ -515,8 +515,8 @@ Fields
       static int
       mytype_traverse(mytype* self, visitproc visit, void* arg)
       {
-          Py_VISIT(self->mt_base);
-          Py_VISIT(self->mt_object_member);
+          Py_VISIT(self->mt_object_member);
```

Comment:
what is the whitespace change here, is it a tab?

Reply:
To be honest, I'm not sure. I did this on mobile, where editing the code directly seemed easier than opening an issue. I'll see the PR on a desktop in ~24 hrs and fix it. In the meantime, feel free to do with it as you wish.